doc/source/BuildingRunningTesting/BuildRunLandDA.rst

```diff
@@ -245,7 +282,7 @@ The experiment has successfully completed when all tasks say SUCCEEDED under STA
 Run Without Rocoto
```

Comment:
Does `run_without_rocoto.sh` no longer work?

Reply:
@gspetro-NOAA, it might work only for a single-day (one cycle) run. Since the sample configurations for WE2E are for two-cycle runs, this script was removed by the previous PR.

Reply:
Ok, I removed the section!